### PR TITLE
Docs: 6.5 update

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
- Updates index page to point at What's new in 6.5
- Updates What's new in 6.5 article

### CODE REVIEW

**Review by RichiH — COMMENTED** [review]
Teeny-tiny change.

File: docs/sources/guides/whats-new-in-v6-5.md [hunk 1]
```diff
@@ -120,18 +160,48 @@ Derived fields allow any part of a log message to be turned into a link. Leaning
...
 
 {{< docs-imagebox img="/img/docs/v65/explore_time_sync.gif" caption="Time-sync of split views in Explore" >}}
 
+### Alerting support for Azure Application Insights
+
+The [Azure Monitor](/features/datasources/azuremonitor/) data source supports multiple services in the Azure cloud. Before Grafana v6.5, only the Azure Monitor service had support for [Grafana Alerting](/alerting/rules). In Grafana 6.5, alerting support has been implemented for the [Application Insights service](/features/datasources/azuremonitor/#querying-the-application-insights-service).
+
+### Allow saving of provisioned dashboards from UI
+
+Historically it has been possible to make changes to a provisioned dashboard in the Grafana UI. However, it hasn't been possible to save the changes without manual intervention. In Grafana 6.5 we introduce a new dashboard provisioning setting named `allowUiUpdates`. If `allowUiUpdates` is set to `true` and you make changes to a provisioned dashboard, you can Save the dashboard and the changes will be persisted to the Grafana database.
```
Comment (aknuds1) [inline docs/sources/guides/whats-new-in-v6-5.md]: s/Save the dashboard/save the dashboard/

File: docs/sources/guides/whats-new-in-v6-5.md [hunk 2]
```diff
@@ -120,18 +160,48 @@ Derived fields allow any part of a log message to be turned into a link. Leaning
...
+
+### Generic OAuth role mapping
+
+Grafana 6.5 makes it possible to configure Generic OAuth to map a certain response from OAuth provider to a certain Grafana organization role, similar to the existing [LDAP Group Mappings](/auth/ldap/#group-mappings) feature. The new setting is named `role_attribute_path` and expects a [JMESPath](http://jmespath.org/) expression.
+
+Read more about this new feature in [Generic OAuth Authentication](/auth/generic-oauth/) and make sure to check out the [JMESPath examples](/auth/generic-oauth/#jmespath-examples).
+
+### Image renderer plugin
+
+Since we announced the deprecation of PhantomJS and new [Image Renderer Plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) in Grafana [6.4](/guides/whats-new-in-v6-4/#phantomjs-deprecation) we’ve received bug reports and valuable feedback.
```
Comment (aknuds1) [inline docs/sources/guides/whats-new-in-v6-5.md]: "and the new [Image Renderer Plugin](https://grafana.com/grafana/plugins/grafana-image-renderer)"

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Spotted a couple of language niggles.

**Review by aknuds1 — APPROVED** [review]
LGTM

**Pull request merged**